js/cobinhood.js

```diff
@@ -500,9 +500,11 @@ module.exports = class cobinhood extends Exchange {
         headers = {};
         if (api === 'private') {
             this.checkRequiredCredentials ();
-            headers['device_id'] = this.apiKey;
+            // headers['device_id'] = this.apiKey;
             headers['nonce'] = this.nonce ();
-            headers['Authorization'] = this.jwt (query, this.secret);
+            // headers['Authorization'] = this.jwt (query, this.secret);
+            // cobinhood give a new JWT token that must be used into Authorization header
```

Comment:
Hi! ) Quick question: can you, please, elaborate on *"cobinhood give a new JWT token"* – how do you obtain the JWT token from them?

Reply:
Do you mean write a better comment into the code?

Reply:
No, I mean, just answer the question ) Afaik, JWT is meant to be precomputed on client side with a secret key depending on the type of encryption used. How do you get the JWT token from cobinhood (because you say, they "give a new token")?

Reply:
Ok, I don't how it worked earlier, but now (if I am doing right) when you login into your account you can generate a new apikey and you get a JWT token (this token is tagged as apikey). If I put this token directly into Authorization header, the request is accepted. Maybe I am doing something wrong, because this way I am not signing every request, and the same token is used with many requests, but I can get balance succesfully. Do you think I am doing something wrong?

Reply:
> Do you think I am doing something wrong?

Nope, not at all, and thank you for explaining it, it's very clear now. So they issue an apiKey on their site and don't use the JWT at all, because of that. You're right, you're using the same token, however, it comes with a nonce number which is still unique per each request. Though, this kind of basic Authorization header is less restrictive, it is as strong as is the underlying HTTPS transport. Cannot use it with public proxies.

Reply:
Hi, I have found a couple of errors:
- when placing an order, size & price parameters must be strings: https://cobinhood.github.io/api-public/#place-order
- in fetchOrderTrades method: when invoke parseTrades without market parameter makes parseTrade fails. So symbol parameter in this function could not be undefined.

Reply:
@lfern no worries about those errors, I am getting the same and I am fixing them along the way while merging your PR. Doing the extensive testing as well. Thx for the notice!

Reply:
> Though, this kind of basic Authorization header is less restrictive, it is as strong as is the underlying HTTPS transport. Cannot use it with public proxies.

@kroitor , do you mean a public proxy can really decrypt HTTPS? Well, ok, anyone can decrypt HTTPS given enough resources but still... is it easier for them to decrypt then for say, your internet provider?

Reply:
> do you mean a public proxy can really decrypt HTTPS?

No, I mean, depending on the type and settings of a public proxy it has to convey SSL-encrypted TCP traffic, like with, say, CORS proxies. And for that, you talk to a proxy directly, instead of talking to an exchange. The proxy decrypts it, then re-encrypts and sends it towards the other party. Thus, because a public proxy is public, you can't totally trust it. Very few proxies actually have a properly configured pass-thru with respect to your privacy.

Reply:
Ah, it's for CORS proxies... never thought about it, now clear, thanks!